tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -526,6 +526,19 @@ module.exports = {
         })
       })
     },
+    getListedFilesFolders: function () {
+      this
+        .waitForElementNotPresent('@filesListProgressBar')
+        .waitForElementVisible({
+          selector: '@allFiles',
+          abortOnFailure: false
```

Comment:
why not to abort if the files are not visible?

Reply:
The test should not fail when there are no files that match the pattern entered for filtering files